src/compas_ags/ags/graphstatics.py

```diff
@@ -488,72 +541,6 @@ def force_update_from_form(force, form):
         attr['y'] = _xy[index, 1]
 
 
-def force_update_from_constraints(force):
```

Comment:
I forgot to merge main in this branch... this should not be deleted. Will update